lms/static/js/certificates/views/certificate_bulk_whitelist.js

```diff
@@ -31,9 +31,9 @@
             return Backbone.View.extend({
                 el: DOM_SELECTORS.bulk_exception,
                 events: {
-                    'change #browseBtn': 'chooseFile',
+                    'change #browseBtn-bulk-csv': 'chooseFile',
```

Comment:
Beyond the scope of this PR. A revamp of the dashboard is forthcoming and can be visited then.